docs/monitoring/autodiscovery.md

```diff
@@ -0,0 +1,60 @@
+title: Sematext Autodiscovery Monitoring
+description: Sematext Autodiscovery Monitoring automatically scans for services that can be monitored using Sematext Agent and, depending on your configuration, automatically sets up monitoring agents. 
+
+## Autodiscovery Monitoring
+
+The centerpiece of Autodiscovery feature is Sematext Agent which scans for services that can be monitoring by Sematext supported integrations. It works in both bare-metal/virtual machine and in container environments. The data about such discovered services is sent to Sematext backend and displayed in UI under Discovery option in the menu on the left.
+
+<img class="content-modal-image" alt="Sematext Autodiscovery UI" src="../images/monitoring/autodiscovery-ui.png" title="Sematext Autodiscovery UI">
+
+
+Autodiscovered services can be monitored in two ways:
+- **Manually** - in case of bare-metal/virtual machine setups, you can use classic Agent installation instructions known from the past. In container environments you can just add `MONITORING_TOKEN` as environment variable to your container and Sematext Agent will automatically match it to the type of discovered service and set up monitoring agent container specifically for that service. In some cases you'll also have to provide credentials (like MySQL username/password) as environment variable or in case of Kubernetes workloads as a secret. Again, just follow the classic monitoring installation instructions in UI to set this up.
```

Comment:
"known from the past" makes sense to you/us.  To users this has no meaning.  They don't track our instructions. 

In container environments you can just add `MONITORING_TOKEN` as environment variable to your container and Sematext Agent will automatically match it to the type of discovered service and set up monitoring agent container specifically for that service. ==> In container environments you can just add `MONITORING_TOKEN` as environment variable to your container.  Sematext Agent will automatically match it to the type of discovered service and set up monitoring agent container specifically for that service.

In some cases you'll also have to provide credentials (like MySQL username/password) as environment variable or in case of Kubernetes workloads as a secret. => If a service requires authentication for monitoring, you will also need to provide credentials, e.g., as environmental variable with username or password for monitoring MySQL or as a Kubernetes Secret.